src/pages/costModels/costModel/index.tsx

```diff
@@ -6,7 +6,7 @@ import {
   TabContent,
   Title,
 } from '@patternfly/react-core';
-import { ErrorCircleOIcon } from '@patternfly/react-icons';
+import { ErrorCircleOIcon } from '@patternfly/react-icons/dist/js/icons/error-circle-o-icon';
```

Comment:
why are we importing this from 'dist/js' and not from '@patternfly/react-icons'?

Reply:
By including exactly what we need, webpack doesn't need to filter thousands of tokens and icons. Webpack opens and parses thru patternfly's 3k+ files to find what we need. Each time we do that, it adds .5-1 sec to the build.